tests/test_fetch.py

```diff
@@ -28,9 +29,16 @@ def test_fetchrepodate_connectionerror(self):
             fetch_repodata(url)
             assert msg in str(execinfo)
 
+    def test_fetchrepodate_connectionerror(self):
+        with pytest.raises(RuntimeError) as execinfo:
+            url = "http://10.0.0.0/"
```

Comment:
We had changed this to`240.0.0.0`.  I think the PR is probably old (my fault) and you should rebase.